prime-router/src/main/kotlin/cli/FileUtilities.kt

```diff
@@ -0,0 +1,117 @@
+package gov.cdc.prime.router.cli
+
```

Comment:
I created this utilities class to parallel HttpUtilities.

There is a refactoring task not yet done:   I didn't fix the file creation code in ProcessData in the cli to use these new methods yet.  :(

Reply:
I think there's a few places we should abstract out some logic into interfaces. This might be one of those.